test/cipher.go

```diff
@@ -21,25 +24,274 @@ func HashBench(b *testing.B, hash func() hash.Hash) {
 
 // Benchmark a stream cipher.
 func StreamCipherBench(b *testing.B, keylen int,
-			cipher func([]byte) cipher.Stream) {
+	cipher func([]byte) cipher.Stream) {
 	key := make([]byte, keylen)
 	b.SetBytes(1024 * 1024)
 	data := make([]byte, 1024)
 	for i := 0; i < b.N; i++ {
 		c := cipher(key)
 		for j := 0; j < 1024; j++ {
-			c.XORKeyStream(data,data)
+			c.XORKeyStream(data, data)
 		}
 	}
 }
 
 // Benchmark a block cipher operating in counter mode.
+/*
+XXX Broken
 func BlockCipherBench(b *testing.B, keylen int,
```

Comment:
It looks like the Travis build is failing because this function is referenced but its definition is missing?